src/olympia/blocklist/templates/admin/blocklist/widgets/blocks.html

```diff
@@ -29,11 +29,10 @@ <h3>{{ blocks|length|intcomma }} Add-on GUIDs with {{ total_adu|intcomma }} user
               name="changed_version_ids"
               value="{{ version.id }}"
               {% if version.id in widget.value %}checked{% endif %}
-            > {% if is_delete %}Unblock{% else %}Block{% endif %} {{ version.version }}</label>
+            > {% if is_add_change %}Block{% else %}Unblock{% endif %} {{ version.version }} {% if version.blockversion %}({{ version.blockversion.get_soft_display }}){% endif %}</label>
           {% else %}
-            <span title="{% if version.is_blocked %}Blocked{% else %}Not blocked{% endif %}">
-              <!-- Red Hexagonal stop sign for Blocked; Green cirle for not blocked -->
-              {% if version.is_blocked %}&#x1F6D1;{% else %}&#x1F7E2;{% endif %}{{ version.version }}
+            <span>
+              {{ version.version }} ({% if version.is_blocked %}{{ version.blockversion.get_soft_display }}{% else %}&#x1F7E2; Not Blocked{% endif %})
```

Comment:
Can you add a screenshot to the PR showing this with a list of versions?  imo showing it this way is going to make it harder to visually scan through a list of versions than previously.

I.e. it's currently (using `#` and `*` in place of green and red symbols):
```
# 12.5
# 13.2.1.567
* 14.3
# 15
* 16.5399834.0
```
and this change makes it:
```
12.5 (# Not Blocked)
13.2.1.567 (# Not Blocked)
14.3 (* Hard-Block)
15 (# Not Blocked)
16.5399834.0 (^ Soft-Block)
```

Reply:
Here is a deletion of a complex case (I've manually increased the height of the list in devtools to display everything):
![Screenshot 2024-10-24 at 18-07-47 Delete Blocks Django site admin](https://github.com/user-attachments/assets/03b53ff5-208d-4017-9d6f-d81b4d9a8977)

Here is an addition with the same add-on in the same state (again manually increased the height of the list to display everything):
![Screenshot 2024-10-24 at 18-10-25 Block Add-ons Django site admin](https://github.com/user-attachments/assets/3ac6c6e1-d91e-4aa1-8b8c-150c14f21568)


IMHO it's necessary to do _something_ to properly visualize such cases.

Reply:
If @wagnerand is okay with that UX then fine

Reply:
This looks ok to me overall.

Minor suggestion: For the blocked versions, can we rename "Soft-Block" to "Soft-Blocked" (and same for hard), to clarify that this is a state, not an action.

Also, when adding a block, where would we indicate whether it is a soft-block or hard-block? (The screenshot only says "Block".) Is this on the page that isn't part of the screenshot?

Reply:
> This is also why btw I've renamed from "Soft-Blocked" to "Soft-Block", since it refers to a type of block, I figured this would work better in all situations. I'm happy to change it back to "Soft-Blocked" if you prefer though.

That makes sense. My comment was about modifying/deleting a block (not adding a new one, since there are no blocked versions). Here, we should distinguish between action (block) and state (blocked). Does that clarify things?

Reply:
Yes, but I was hoping to re-use the string as it was complicated enough as it is...

Reply:
Going back to the different actions (again those are the most complex cases, on purpose, showing all possible states), I think it should already be clear in all cases:

### Adding a block

![Screenshot 2024-10-29 at 11-43-02 Block Add-ons Django site admin](https://github.com/user-attachments/assets/fdffa3fb-5f0b-4c22-a536-3f0fb6cbe8d0)

You can see this add-on already has a soft and a hard blocked version, with the block type displayed in parenthesis. Since it's an addition, those don't get the checkbox for those. We only use the word "Block" next to the checkboxes, since the type of block is selected for the whole submission by the dropdown below.

(I've cheated and removed the block history in this screenshot as this block has been modified over and over and over and it was taking a bunch of space)

### Approving a pending block

![Screenshot 2024-10-29 at 11-54-38 Pending Sign-off {105ca783-0522-4143-b8ed-062cba7028a9} Change blocklist submission Django site admin](https://github.com/user-attachments/assets/efe402e3-3083-4835-b80d-5a1dba1a90ae)

Same add-on after I submitted one new hard blocked version that needs to be approved. The block type is no longer modifiable. Since this submission only contained a single version to be blocked, that's the only checkbox shown, and again only the word "Block" shows up for it, as the type of block would be determined by the block type for this submission.

### Deleting a block

![Screenshot 2024-10-24 at 18-07-47 Delete Blocks Django site admin](https://github.com/user-attachments/assets/03b53ff5-208d-4017-9d6f-d81b4d9a8977)

This time, there is no global "Block type" for this submission since it's a deletion. We only show the checkboxes next to versions that are soft or hard blocked already. We show the word "Unblock" for those, and again the current block type for each version in parenthesis.

Reply:
I found a way to do what @wagnerand asked for in 4391116d2e3d6a73550798639c1429220c79cd3d.

So every time we display the state of a blocked version it will say "Hard/Soft-Blocked", but for the block type of a submission it will say "Hard/Soft Block".